src/routes/(inner)/api/use-focus/+page.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+	import CodeBlock from '$docs/components/CodeBlock/CodeBlock.svelte';
+	import ExampleRaw from './Example.svelte?raw';
+	import Table from '$docs/components/Table/Table.svelte';
+	import { tableOptions } from './data.js';
+</script>
+
+<div class="space-y-10">
+	<!-- Header -->
+	<header class="card card-gradient space-y-8">
+		<h1 class="h1"><span>useFocus</span></h1>
+		<p>
+			Opens the floating element while the reference element has focus, like CSS
+			<kbd class="kdb">:focus</kbd>.
```

Comment:
This should like be a code element instead of kbd. No visible formatting as is:

![Screenshot 2024-05-16 at 4 29 44 PM](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/75f7baff-be99-48c9-9a15-f6e798ecf8d8)

Reply:
Feel free to add and comment out upcoming linked sections like this:

> NOTE: include a `TODO` comment so we can find it later!

![Screenshot 2024-05-16 at 4 30 53 PM](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/e07c21ab-fcef-471f-acb3-ec54e15be06c)